README.md

```diff
@@ -1,46 +1,89 @@
 
-In the project directory, you can run:
+## Getting Started
 
-### `npm start`
+You can benift of the platform services by going to the following link: [TeamsLounge](https://www.teamslounge.com/)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+### Prerequisites
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+What things you need to install the software and how to install them
 
-### `npm test`
+```
+Give examples
+```
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Installing
 
-### `npm run build`
+A step by step series of examples that tell you how to get a development env running
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Say what the step will be
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+```
+Give the example
+```
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+And repeat
 
-### `npm run eject`
+```
+until finished
+```
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+End with an example of getting some data out of the system or using it for a little demo
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## Running the tests
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+Explain how to run the automated tests for this system
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+### Break down into end to end tests
 
-## Learn More
+Explain what these tests test and why
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+```
+Give an example
+```
+
+### And coding style tests
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+## Deployment
+
+First alpha deployment is coming soon.
+
+## Built With
+
+* [Dropwizard](http://www.dropwizard.io/1.0.2/docs/) - The web framework used
```

Comment:
this section should be fixed

Reply:
please add the icons from this readme file https://github.com/Ibrahimrahhal/Ibrahimrahhal